docs/src/sections/mobileapp_solution.adoc

```diff
@@ -0,0 +1,493 @@
+=== Aplikacja mobilna
+
+Aplikacja mobilna została wykonana z pomocą podobnych technologii i narzędzi jak w przypadku aplikacji mobilnej.
+Podstawę stanowią język programowania TypeScript oraz framework React Native, który jest wersją framework'a react
+dostosowaną do tworzenia natywnych aplikacji mobilnych. Pozwala on na równoczesne tworzenie kodu na platformy
+Android oraz iOS, jednak w projekcie zdecydowano się wspierać tylko system Android. Jest to podyktowane brakiem
+potrzebnych urządzeń funduszy. Tak samo, jak w aplikacji mobilnej, projekt budowany jest przy pomocy programu _yarn_,
```

Comment:
chyba brakiem funduszy na potrzebne urządzenia

Reply:
Miałobyć "urządzeń oraz funduszy". Apple ma chore opłaty za samo tworzenie aplikacji że to sie w głowie nie mieści